operations/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+- include: upgrade-docker.yml
```

Comment:
Do we need `main` playbook here? I was thinking we'd keep on adding more operations to this role - and none of them would be 'main' or 'default'

Maybe `when:` should go to `upgrade-docker.yml` tasks then?

Reply:
Yes, because this is intended to be a subrole.

```
roles:
- osbs-namespace/operation
```

I explicitly wanted to keep this away from this role's main purpose which is to setup namespaces for OSBS usage.

Reply:
Ah, right, agreed